app/Components/CreateProblemModal.js

```diff
@@ -1,34 +1,34 @@
-import  Modal  from "react-native-modal";
+import Modal from "react-native-modal";
 import { Text, View, StyleSheet, Button, TextInput } from "react-native";
 import { useState, useContext } from "react";
-import RNPickerSelect from "react-native-picker-select";
+import { Picker } from '@react-native-picker/picker';
 import RouteContext from "./RouteContext";
 import { useNavigation } from '@react-navigation/native';
 
-const CreateProblemModal = ({isVisible, toggleCreateProblemModal, handlePostProblem}) => {
+const CreateProblemModal = ({ isVisible, toggleCreateProblemModal, handlePostProblem }) => {
   const [problemName, setProblemName] = useState('');
   const [problemGrade, setProblemGrade] = useState('');
   const [nameError, setNameError] = useState("");
   const navigation = useNavigation();
   const { setNewVectors, wallInfo } = useContext(RouteContext);
-  const availableGrades = [
-    { label: "V0", value: "V0" },
-    { label: "V1", value: "V1" },
-    { label: "V2", value: "V2" },
-    { label: "V3", value: "V3" },
-    { label: "V4", value: "V4" },
-    { label: "V5", value: "V5" },
-    { label: "V6", value: "V6" },
-    { label: "V7", value: "V7" },
-    { label: "V8", value: "V8" },
-    { label: "V9", value: "V9" },
-    { label: "V10", value: "V10" },
-    { label: "V11", value: "V11" },
-    { label: "V12", value: "V12" },
-    { label: "V13", value: "V13" },
-    { label: "V14", value: "V14" },
-    { label: "V15", value: "V15" },
-];
+  // const availableGrades = [
+  //   { label: "V0", value: "V0" },
+  //   { label: "V1", value: "V1" },
+  //   { label: "V2", value: "V2" },
+  //   { label: "V3", value: "V3" },
+  //   { label: "V4", value: "V4" },
+  //   { label: "V5", value: "V5" },
+  //   { label: "V6", value: "V6" },
+  //   { label: "V7", value: "V7" },
+  //   { label: "V8", value: "V8" },
+  //   { label: "V9", value: "V9" },
+  //   { label: "V10", value: "V10" },
+  //   { label: "V11", value: "V11" },
+  //   { label: "V12", value: "V12" },
+  //   { label: "V13", value: "V13" },
+  //   { label: "V14", value: "V14" },
+  //   { label: "V15", value: "V15" },
+  // ];
```

Comment:
MP: seems that we got some comments left out here, but we can definitely fix this in the next branch too!

Reply:
Ooof totally left this, maybe whoever adds the post test can delete this too?? Or if I end up doing that I can delete it?

Reply:
I can take care of that